plugins/nlp.h

```diff
@@ -30,24 +29,31 @@ struct ApplyStopWordsFunctionConfig {
 
 DECLARE_STRUCTURE_DESCRIPTION(ApplyStopWordsFunctionConfig);
 
+struct Words {
```

Comment:
This feels a bit strange.  Could a typedef for Words and Documenr be used instead?  If not, may be we can renames
```
struct Words {
   ExpressionValue counts;
}
```
and
```
struct Document {
    ExpressionValue text;
}
```